tests/Properties.hs

```diff
@@ -726,6 +726,7 @@ prop_takeWhile f xs = L.takeWhile f (pack xs) == pack (takeWhile f xs)
 prop_dropWhile f xs = L.dropWhile f (pack xs) == pack (dropWhile f xs)
 prop_takeWhileEnd f = P.takeWhileEnd f `eq1` (P.reverse . P.takeWhile f . P.reverse)
 prop_dropWhileEnd f = P.dropWhileEnd f `eq1` (P.reverse . P.dropWhile f . P.reverse)
+prop_dropEnd i xs = L.dropEnd (fromIntegral i) (pack xs) == pack (take (length xs - i) xs)
```

Comment:
`L.dropEnd` is not in scope, as witnessed by CI: https://travis-ci.org/github/haskell/bytestring/jobs/731664372#L645
Should be `P.dropEnd`, I suppose.